src/ax/marshaling/message_test.go

```diff
@@ -32,49 +33,102 @@ var _ = Describe("MarshalMessage", func() {
 })
 
 var _ = Describe("UnmarshalMessage", func() {
+
 	message := &messagetest.Message{
 		Value: "<value>",
 	}
-	data, err := proto.Marshal(message)
+
+	nonaxmsg := &messagetest.NonAxMessage{}
+
+	pbdata, err := proto.Marshal(message)
 	if err != nil {
 		panic(err)
 	}
 
-	It("unmarshals the message using the protocol specified in the content-type", func() {
-		m, err := UnmarshalMessage(
-			"application/vnd.google.protobuf; proto=ax.internal.messagetest.Message",
-			data,
-		)
-		Expect(err).ShouldNot(HaveOccurred())
-		Expect(proto.Equal(m, message)).To(BeTrue())
-	})
-
-	It("returns an error if the content-type is invalid", func() {
-		_, err := UnmarshalMessage("", data)
-		Expect(err).Should(HaveOccurred())
-	})
+	_, jsondata, err := MarshalJSON(message)
```

Comment:
This should use the underlying PBJSON marshaller, so that we verify our output against that, otherwise we are just comparing our own function to itself